packages/react-searchbox/src/components/SearchBox.js

```diff
@@ -439,6 +489,196 @@ class SearchBox extends React.Component {
     }
   };
 
+  onKeyDown = e => {
+    if (isEmpty(this.props.focusShortcuts)) {
+      return;
+    }
+
+    // for hotkeys' combinations such as 'cmd+k', 'ctrl+shft+a', etc, we use hotkeys-js
+    if (this.hotKeyCombinationsUsed) {
+      this.hotkeys(
+        parseFocusShortcuts(this.props.focusShortcuts).join(','),
+        (event, handler) => {
+          // Prevent the default refresh event under WINDOWS system
+          event.preventDefault();
+          const elt = e.target || e.srcElement;
+          const tagName = elt.tagName;
+          if (elt.isContentEditable || tagName === 'INPUT') {
+            // already in an input
+            return;
+          }
+          this.searchInputField.current.focus();
+        }
+      );
+      return;
+    }
+    const shortcuts = this.props.focusShortcuts.map(key => {
+      if (typeof key === 'string') {
+        return isNumeric(key) ? +key : key.toUpperCase().charCodeAt(0);
+      }
+      return key;
+    });
+    const elt = e.target || e.srcElement;
+    const tagName = elt.tagName;
+    if (elt.isContentEditable || tagName === 'INPUT') {
+      // already in an input
+      return;
+    }
+    let which = e.which || e.keyCode;
```

Comment:
https://stackoverflow.com/questions/29811465/why-does-the-slash-key-not-have-an-ascii-value-of-47/29811987 
give ref to make it user understandable